scripts/src/release/releasebody.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+sys.path.append('./scripts/src/')
+from release import releasechecker
+from utils import utils
+
+def make_release_body(version, image_name, release_info):
```

Comment:
Since we're making modifications here (even if we're just moving this), could I have you add docstrings?